### PR TITLE
Better CLI Help Messages

### PR DESCRIPTION
**Pull request by YrrepNoj**
Improve the CLI `--help` message outputs.

### CODE REVIEW

File: src/cmd/prepare.go [hunk 1]
```diff
@@ -72,7 +72,8 @@ var prepareComputeFileSha256sum = &cobra.Command{
 var prepareFindImages = &cobra.Command{
 	Use:     "find-images",
 	Aliases: []string{"prep"},
-	Short:   "evaluates components in a zarf file to identify images specified in their helm charts and manifests",
+	Short: "Evaluates components in a zarf file to identify images specified in their helm charts and manifests.\n" +
+		"Requires the `--repo-chart-path` flag to be provided.",
```
Comment (jeff-mccoy) [inline src/cmd/prepare.go:76]: this flag is optional, but needed for big bang and should be "-p ./chart".  Technically could be for any helm chart hosted in git, but we focus on Big Bang

File: src/cmd/initialize.go [hunk 2]
```diff
@@ -46,8 +52,7 @@ var initCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(initCmd)
 	initCmd.Flags().BoolVar(&config.DeployOptions.Confirm, "confirm", false, "Confirm the install without prompting")
-	initCmd.Flags().StringVar(&config.DeployOptions.Components, "components", "", "Comma-separated list of components to install.  Adding this flag will skip the init prompts for which components to install")
-
+	initCmd.Flags().StringVar(&config.DeployOptions.Components, "components", "", "Comma-separated list of components to instal.")
```
Comment (jeff-mccoy) [inline src/cmd/initialize.go:55]: instal -> install

File: src/cmd/initialize.go [hunk 3]
```diff
@@ -18,8 +18,14 @@ import (
 var initCmd = &cobra.Command{
 	Use:     "init",
 	Aliases: []string{"i"},
-	Short:   "Deploys the gitops service or appliance cluster on a clean linux box",
-	Long:    "Flags are only required if running via automation, otherwise the init command will prompt you for your configuration choices",
+	Short:   "Prepares a k8s cluster for the deployment of Zarf packages",
+	Long: "Uses a local 'zarf-init' package found either in the directory you executed " +
+		"the command from or the directory that the Zarf binaries exists. " +
```
Comment (jeff-mccoy) [inline src/cmd/initialize.go:23]: "the command from or the directory" reads a little odd I think

**Review by jeff-mccoy — CHANGES_REQUESTED** [review]

File: src/cmd/prepare.go (hunk 1 above)
Comment (YrrepNoj) [reply src/cmd/prepare.go]: Thanks, that makes sense. I updated the message (as well as corrected the message type).

**Review by jeff-mccoy — APPROVED** [review]

**Pull request merged**